pkg/controller/account/account_controller.go

```diff
@@ -442,14 +422,23 @@ func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Resul
 		if err != nil {
 			return reconcile.Result{}, err
 		}
+		currentAcctInstance.Status.RotateCredentials = false
 	}
 	if accountNeedsConsoleCredentialsRotated(currentAcctInstance) {
 		reqLogger.Info(fmt.Sprintf("rotating console URL credentials for %s", currentAcctInstance.Name))
 		err = r.RotateConsoleCredentials(reqLogger, awsSetupClient, currentAcctInstance)
 		if err != nil {
 			return reconcile.Result{}, err
 		}
+		currentAcctInstance.Status.RotateConsoleCredentials = false
 	}
+
+	// Push the updated account to the server.
+	err = r.Client.Status().Update(context.TODO(), currentAcctInstance)
```

Comment:
So now this will always push a (single) status update. But I think there are still code paths where we didn't actually rotate anything and the update would be a no-op, right? That may not be a big deal, but if you wanted to keep the `credentialsRotated` bool from the previous iteration, you could get rid of that.

Reply:
There might be, but this is also where the account creation gets pushed, as part of this refactor was forcing the credentials to be rotated/created when the account is created as well as when the CR is updated so that the credentials get rotated.

With that said, though, the "credential rotation" will be getting refactored right after this PR in another one to move that functionality out of the CR and just into the secret_watcher outright.  We'll still need to create the users and secrets here, but then we'd circle back on whether we want to do one update at the end even if nothing's changed compared to specifically updating when something changes.

Reply:
Okay, sounds like a plan. Thanks!

/lgtm